libafl_bolts/src/llmp.rs

```diff
@@ -1775,6 +1775,21 @@ where
         }
     }
 
+    /// Receive the buffer, also reading the LLMP internal message flags
+    #[allow(clippy::type_complexity)]
+    #[inline]
+    pub fn recv_buf_blocking_with_flags(&mut self) -> Result<(ClientId, Tag, Flags, &[u8]), Error> {
```

Comment:
Instead of allowing type_complexity, maybe it's time for a struct?

Reply:
Sure, let's include that change in #2068?